index.html

```diff
@@ -53,211 +53,92 @@
     <title>NEW DEVS ON THE BLOCK</title>
   </head>
   <body>
-    <header>
+    <header id="top">
+      <nav>
+        <a href="#about">About</a>
+        <a href="#schedule">Schedule</a>
+        <a href="#workshops">Workshops</a>
+        <a href="#coaches">Coaches</a>
+      </nav>
       <img
         src="images/logo.png"
         class="logo"
         alt="kickass new devs on the block logo"
       />
     </header>
     <main>
-      <section>
-        <h2>Become a NEW DEV on the Block</h2>
+      <article id="about">
+        <h2>About us</h2>
```

Comment:
We should decide if we want this section to be written directly to potential participants or rather more generally. 

For example, if we want to write directly to people considering attending our workshop then we should maybe consider rearranging it a bit (focusing on the 'guiding you through coding your first website' part first).

Reply:
Actually after thinking about it, I like the version that's currently on the website a bit better 🤔  Maybe we can find a fusion between the two?

Reply:
How do you like the updated version @carolstran ?